CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Update card components to make them more consistent [#1383](https://github.com/opendatateam/udata/pull/1383) [#1460](https://github.com/opendatateam/udata/pull/1460)
 - Switch from OAuthlib/Flask-OUAhtlib to Authlib and support all grants type as well as token revocation [#1434](https://github.com/opendatateam/udata/pull/1434)
 - Fix completion/suggestion unicode handling [#1452](https://github.com/opendatateam/udata/pull/1452)
+- udata is now protocol (`http`/`https`) agnostic. This is now fully the reverse-proxy responsibility [#1463](https://github.com/opendatateam/udata/pull/1463)
```

Comment:
Maybe add a `breaking`? People have to make sure they configure their reverse proxy correctly, or their login data will travel unencrypted AFAIK.